### PR TITLE
data: MuhammadRabi

### PR DESCRIPTION
**Pull request by MuhammadRabi**
Check List (Check all the applicable boxes)
[ ✔] My code follows the code style of this project.
[ ✔] My change requires changes to the documentation.
[ ✔] I have updated the documentation accordingly.
[ ✔] All new and existing tests passed.
[ ✔] This PR does not contain plagiarized content.
[✔ ] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6971"></a>

### CODE REVIEW

**Review by Aniket1026 — APPROVED** [review]
Looks good to merge ✨✨

File: data/MuhammadRabi.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Muhammad Rabi",
+  "type": "personal",
+  "bio": "I'm an Autodidact Front-end web developer, seeking contribution to open source projects",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/MuhammadRabi",
+      "icon": "FaGitHub"
```
Comment (kumarsonsoff3) [inline data/MuhammadRabi.json:9]: You're probably using an invalid icon here. It should be `FaGithub`

```suggestion
      "icon": "FaGithub"
```

File: data/MuhammadRabi.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/muhammad-rabi-07a51b202/",
+      "icon": "FaLinkedIn"
```
Comment (kumarsonsoff3) [inline data/MuhammadRabi.json:14]: Same here. It should be `FaLinkedin`

```suggestion
      "icon": "FaLinkedin"
```

Comment (MuhammadRabi) [reply data/MuhammadRabi.json:14]: Thank you for your help

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**